### PR TITLE
Fix inconsistencies

### PR DESCRIPTION
**Pull request by justmobilize**
- Add ipv4_address to match native wifi and esp32spi
- Fix recvfrom_into to return the correct address info

### CODE REVIEW

**Comment by justmobilize** [comment]
@anecdata would appreciate your eyes as well

**Review by anecdata — COMMENTED** [review]
LGTM! Tested on PicoW + WIZnet 5100S Pico hat. Client sends, server receives then sends, client receives... rinse, repeat.

File: adafruit_wiznet5k/adafruit_wiznet5k.py [hunk 1]
```diff
@@ -336,6 +336,15 @@ def ip_address(self) -> bytes:
         """
         return self._read(_REG_SIPR[self._chip_type], 0x00, 4)
 
+    @property
+    def ipv4_address(self) -> bytes:
```
Comment (anecdata) [inline adafruit_wiznet5k/adafruit_wiznet5k.py:340]: should this show return is string?

Comment (justmobilize) [reply adafruit_wiznet5k/adafruit_wiznet5k.py]: Fixed! 75ee080404c6218667dd8b320992d284ecb114b4

**Comment by justmobilize** [comment]
@dhalbert any concerns with this?

**Review by dhalbert — APPROVED** [review]
:+1:

**Pull request merged**